lib/index.js

```diff
@@ -25,15 +25,15 @@ const bindingsPath = getBindingsPath(rootDir);
 
 function prepareBindings(bindings) {
   bindings.SecretKey.prototype.toHex = function toHex() {
-    return `0x${this.serialize().toString("hex")}`;
+    return `0x${Buffer.from(this.serialize().buffer).toString("hex")}`;
```

Comment:
Is there ever the case where the underlying memory / ArrayBuffer might be shared?

Might be good to specify the offset and length used by the view, to avoid issues like https://github.com/fastify/fastify/issues/5118

```js
Buffer.from(uint8.buffer, uint8.byteOffset, uint8.byteLength).toString("hex")
```

Reply:
Should not be but ill add just in case... Updated.